File: Data Structures & Algorithms/maximum-subarray/submission-0.java
class Solution {
    public int maxSubArray(int[] nums) {
        int sums=0;
        int maxs=Integer.MIN_VALUE;

        for (int x: nums){
            sums=sums+x;
            maxs=Math.max(maxs,sums);
            if (sums<0){
                sums=0;
            }
        }
        return maxs;
        
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums=set(nums)
        
        max_length=0
        for i in nums:
            if i-1 in nums:
                continue
            else:
                current=i
                length=1
            while current+1 in nums:
                current=current+1
                length+=1

            max_length=max(length,max_length)

        return max_length


        